committeeoversightapp/templates/list.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="hearing-form card">
+
+ <p><a href="/" class="btn btn-success">Create new hearing</a></p>
+
+ <table class="table table-striped" id="hearing-table">
+  <thead>
+    <tr>
+      <th scope="col" style="width: 60%">Hearing Title</th>
```

Comment:
This is normally how I set table column width, too! But I was curious if Bootstrap (4) had a way to accomplish the same thing. Yes, it does: https://stackoverflow.com/questions/37924104/table-column-sizing

Reply:
elegant!